src/_shared/scss/_brandbadge.scss

```diff
@@ -0,0 +1,145 @@
+.badge {
```

Comment:
Do you mean `adverts__badge`? I can't find a use of `advert__badge` in the codebase...